Sources/SwifQL/Functions.swift

```diff
@@ -849,15 +849,37 @@ extension Fn {
     
     /// Returns JSON value pointed to by path_elems (equivalent to #> operator)
     /// [Learn more →](https://www.postgresql.org/docs/current/functions-json.html)
-//    public static func json_extract_path(from_json : SwifQLable, path_elems: [String]) -> SwifQLable { // TBD
-//        return _buildFn(.json_extract_path, body: aggregateExpression.parts)
-//    }
+    public static func json_extract_path(_ from_json: SwifQLable, path_elems: [String]) -> SwifQLable {
```

Comment:
Would be great to have aliases to all these new functions with variadic like this
```swift
public static func json_extract_path(_ from_json: SwifQLable, path_elems: String...) -> SwifQLable {
    json_extract_path(from_json, path_elems: path_elems)
}

public static func json_extract_path(_ from_json: SwifQLable, path_elems: [String]) -> SwifQLable {
  ...
}
```

Reply:
I will add 👍